packages/kolibri-common/composables/useBaseSearch.js

```diff
@@ -278,12 +280,19 @@ export default function useBaseSearch({
       if (get(isUserLoggedIn)) {
         fetchContentNodeProgress?.(getParams);
       }
+
+      const query = get(route).query;
+      if (query?.filter_quiz) {
+        getParams.contains_quiz = true;
```

Comment:
This is still not working well, I am now able to see non-exercise resources in search.

![image](https://github.com/user-attachments/assets/c451d0f7-8683-4a05-a483-3f1c6144b15d)

Again, I dont think the route query should be the way to do this. We can cohack to find a better solution!